src/Collector.cs

```diff
@@ -35,6 +35,10 @@ class Collector
     private bool _queueFullWarned;
     private static readonly ILog _log = LogManager.GetCurrentClassLogger();
 
+    private static readonly string InstrumentalAddress = "collector.instrumentalapp.com";
+    private static readonly int InstrumentalPort = 8000;
+    private static readonly byte[] InstrumentalOk = {111, 107, 10};
```

Comment:
How about `Encoding.ASCII.GetBytes("ok\n")` instead of `{111, 107, 10}`. I think it makes the code more obvious.